frontend/src/app/forms/steps/pictures-entry/pictures-entry.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input } from '@angular/core';
+import { /*DomSanitizer,*/ SafeResourceUrl } from '@angular/platform-browser';
+import { CameraResultType, CameraSource, Plugins } from '@capacitor/core';
+
+import { FormGroup } from '../../../common/forms/forms';
+import { FormsData } from '../../../model/forms-data.model';
+
+@Component({
+  selector: 'form-pictures-entry',
+  templateUrl: './pictures-entry.component.html',
+  styleUrls: ['./pictures-entry.component.scss'],
+})
+
+/*
+  To save Files as Data.url just change restultType and uncomment everything in this file
+*/
+export class PicturesEntryComponent {
+  constructor(/*private sanitizer: DomSanitizer*/) {}
+  @Input()
+  formsData: FormGroup<FormsData>;
+
+  photo: SafeResourceUrl;
+
+  async takePicture(): Promise<void> {
+    const image = await Plugins.Camera.getPhoto({
+      quality: 100,
+      allowEditing: false,
```

Comment:
I am curious - what does "allowEditing" do if we enable it? It probably doesn't do anything when previewing in the browser. But in our real app, would this display some sort of cropping controls?

Reply:
I didn't test it, but i did read that in the mobile phone, you should then be able to edit the photo after you have taken it. Which edit functions we then have, needs to be tested. Also there is a correlation, if you want to save the picture in the photo gallery of the mobile phone. For this i think you need to allow editing then.

Reply:
>Also there is a correlation, if you want to save the picture in the photo gallery of the mobile phone. For this i think you need to allow editing then.

In the Capacitor docs, it says 

> 
> // Whether to save the photo to the gallery. If the photo was picked from the gallery, it will only be saved if edited. Default: false
> saveToGallery ?: boolean;

I think that rather means: If you allow users to pick a photo **from** the gallery. Then the edited photo will only be saved again to the gallery if some changes were made to it. So this doesn't apply to our use case imho.

In general: Saving the photo to the gallery isn't something we should do (data protection concerns, etc.). But it would be cool if editing would be possible. So, maybe, we could check that out once we have actually built the app for iOS. :)